applications/asset_tracker_v2/src/modules/gnss_module.c

```diff
@@ -454,6 +457,37 @@ static void search_start(void)
 		}
 	}
 
+#if defined(CONFIG_GNSS_MODULE_ELEVATION_MASK)
+	err = nrf_modem_gnss_elevation_threshold_set(CONFIG_GNSS_MODULE_ELEVATION_MASK);
+	if (err) {
+		LOG_ERR("Failed to set elevation threshold to %d: %d",
+			CONFIG_GNSS_MODULE_ELEVATION_MASK, err);
+	} else {
+		LOG_DBG("Set elevation threshold");
+	}
+#endif
+
+	uint8_t use_case = NRF_MODEM_GNSS_USE_CASE_MULTIPLE_HOT_START;
+
+	if (IS_ENABLED(CONFIG_GNSS_MODULE_DISABLE_SCHEDULED_DOWNLOADS)) {
+		/* prevent modem from wasting time downloading missing ephemerides */
+		use_case |= NRF_MODEM_GNSS_USE_CASE_SCHED_DOWNLOAD_DISABLE;
+		err = nrf_modem_gnss_use_case_set(use_case);
+		if (!err) {
+			LOG_DBG("Disabled scheduled GNSS downloads");
+		} else {
+			LOG_DBG("Could not disable scheduled GNSS downloads: %d", err);
+			use_case &= ~NRF_MODEM_GNSS_USE_CASE_SCHED_DOWNLOAD_DISABLE;
+			err = nrf_modem_gnss_use_case_set(use_case);
+		}
+	} else {
+		err = nrf_modem_gnss_use_case_set(use_case);
+	}
+	if (err) {
+		LOG_ERR("Failed to set GPS use case, error: %d", err);
+		return;
+	}
```

Comment:
I just now realized, that this isn't actually relevant with ATv2. ATv2 uses GNSS in single fix mode and scheduled downloads are only used in periodic mode. The use case configuration here can just be removed.

Reply:
OK, if there are no plans to support periodic mode in ATv2 (@simensrostad ?)  I'll remove it.

Reply:
There are no immediate plans of supporting this. @jtguggedal

Reply:
My reluctance to remove it is because it would be possible for a customer to change the mode, in which case this would be required if they enable filtered ephemerides. But I will remove it since you all are in agreement with each other.